File: Leetcode/171-Excel-Sheet-Column-Number/ExcelSheetColumnNumber.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int len = s.length();
        if (len < 1) {
            return 0;
        }
        int total = 0;
        
        for (int i=0; i<len; i++) {
            total *= 26;
            total += s.at(i)-'A'+1;
        }
        return total;
    }
};

File: Leetcode/171-Excel-Sheet-Column-Number/ExcelSheetColumnNumber.go
func titleToNumber(s string) int {
	length := len(s)
	if length < 1 {
		return 0
	}
	total := 0
	i := 0
	for i < length {
		total *= 26
		total += int(s[i]) - int('A') + 1
		i++
	}
	return total
}
